nemo_skills/pipeline/utils.py

```diff
@@ -294,16 +295,23 @@ def cluster_download(tunnel: SSHTunnel, remote_dir: str, local_dir: str, remote_
     # Get the directory size
     result = tunnel.run(f'du -sb {remote_dir} | cut -f1')
     total_size = int(result.stdout.strip())
-    # Command for streaming the compression progress
-    command = (
-        f'cd {remote_dir_parent} && '
-        f'tar -cf - {remote_dir_name} | '
-        f'pv -s {total_size} -p -t -e -b -F "Compressing Remote Directory: %b %t %p" | '
-        f'gzip > {remote_tar}'
-    )
 
-    # Run the remote compression command and stream the progress
-    result = tunnel.run(command, watchers=[OutputWatcher()], pty=True, hide=False)
+    # Certain systems may not have the `pv` command
+    streaming_possible = shutil.which("pv") is not None
```

Comment:
It's the cluster that didn't have pv, not the local machine. This check is just for local right ?